src/Schema/Blueprint.php

```diff
@@ -127,7 +127,7 @@ public function uuid($column = 'uuid')
      * @param int|null $length
      * @return ColumnDefinition
      */
-    public function binary($column, $length = null)
+    public function binary($column, $length = null, $fixed = false)
```

Comment:
You should add the type specification of the added arguments to the Doc.

Reply:
Fixed. Thanks.